perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -360,7 +360,17 @@ sub inspect : Private {
             }
             if ( $problem->state ne 'duplicate' ) {
                 $problem->unset_extra_metadata('duplicate_of');
+            } else {
+                my $duplicate_of = $c->get_param('duplicate_of');
+                $problem->set_extra_metadata( duplicate_of => $duplicate_of );
+                my $dupe = $c->model('DB::Problem')->find($duplicate_of);
```

Comment:
If you mark a report as a duplicate but don't select a report then this causes a 500 error so there should probably be something to catch that before we get to here.